src/routers/sessions.py

```
@@ -448,27 +449,27 @@ async def get_session_context(
 
 @router.post(
     "/{session_id}/search",
-    response_model=Page[schemas.Message],
+    response_model=list[schemas.Message],
     dependencies=[
         Depends(require_auth(workspace_name="workspace_id", session_name="session_id"))
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)